src/lib/CRSF2MSP/crsf2msp.h

```diff
@@ -1,44 +1,39 @@
 #pragma once
 
-#include <cstdint>
-#include "FIFO.h"
-#include "crsfmsp_common.h"
+#include <functional>
+
 #include "crc.h"
-#include "logging.h"
+#include "crsfmsp_common.h"
 
 /*  Takes a CRSF(MSP) frame and converts it to raw MSP frame
     adding the MSP header and checksum. Handles chunked MSP messages.
 */
 
-class CROSSFIRE2MSP
+class CROSSFIRE2MSP final
 {
+public:
+    CROSSFIRE2MSP() = default;
```

Comment:
Why explicit constructor with default? Forcing a user-declared constructor makes the class non-aggregate which doesn't hurt how we are using it, but what does this add?